server/src/modules/posts/post.entity.ts

```diff
@@ -15,7 +15,8 @@ export class Post {
   description: string;
   // @Prop({ required: true })
   // type: string;
-
+  @Prop({ required: true })
```

Comment:
why do we need active musicians for the post? it's a property from the ensemble, it's not something that we need when creating a post

Reply:
Yeah you are right I mixed it up, I removed it now